### PR TITLE
feat: doc enhancements for toolbox (OSDOCS-1508)

### PR DESCRIPTION
**Pull request by mikemckiernan**
- Overview of toolbox
- Install packages
- Start an alternative image

### CODE REVIEW

**Comment by mikemckiernan** [comment]
@miabbott, this is the PR for the content that you reviewed.  Others are invited to comment.

/hold

**Comment by miabbott** [comment]
LGTM.

cc: @yuqi-zhang @mike-nguyen you folks have dabbled with `toolbox` before

**Comment by mike-nguyen** [comment]
LGTM too

**Comment by mikemckiernan** [comment]
@openshift/team-documentation please review

/hold cancel

File: modules/about-toolbox.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="about-toolbox_{context}"]
+= About `toolbox`
+
+`toolbox` is a tool that starts a container on a {op-system-first} system.
+The tool is primarily used to start a container that includes the required binaries and plug-ins that are needed to run commands such as `sosreport` and `redhat-support-tool`.
```
Comment (codyhoag) [inline modules/about-toolbox.adoc:9]: We generally avoid hard wrapping. There are several places in this PR that break up lines at sentence endings. I personally think hard wrapping at sentence ends mostly mitigates the disadvantages of hard wrapping, but I think the team has taken the stance of discouraging hard wrapping in general.

File: modules/support-installing-packages-to-a-toolbox-container.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="installing-packages-to-a-toolbox-container_{context}"]
+= Installing packages to a `toolbox` container
+
+By default, running the `toolbox` command starts a container with the `registry.redhat.io/rhel8/support-tools:latest` image.
+This image contains the most frequently used support tools.
+If you need to collect node-specific data that requires a support tool that is not part of the image, then you can install additional packages.
```
Comment (codyhoag) [inline modules/support-installing-packages-to-a-toolbox-container.adoc:10]: ISG discourages using if-then in sentence structures. 

> ! if...then conj
Avoid using “then” to introduce an independent clause that follows an “if” clause. For example, write “If
you set a password, access is restricted,” not “If you set a password, then access is restricted.”

File: support/gathering-cluster-data.adoc [hunk 3]
```diff
@@ -43,3 +43,14 @@ include::modules/support-collecting-network-trace.adoc[leveloffset=+1]
 
 // Providing diagnostic data to Red Hat Support
 include::modules/support-providing-diagnostic-data-to-red-hat.adoc[leveloffset=+1]
+
+// About `toolbox`
+include::modules/about-toolbox.adoc[leveloffset=+1]
+
+// Installing packages to a toolbox container
+[discrete]
```
Comment (codyhoag) [inline support/gathering-cluster-data.adoc:51]: More of a question/opinion: why are the proceeding modules marked as `[discrete]`? IMHO, I found them to be informative and beneficial to be highlighted in the TOC for `toolbox`. I'll leave the choice up to you, but figured I'd flag it for consideration. :slightly_smiling_face:

**Review by codyhoag — COMMENTED** [review]
Just a couple minor suggestions; LGTM otherwise!

File: modules/about-toolbox.adoc (hunk 1 above)
Comment (mikemckiernan) [reply modules/about-toolbox.adoc:9]: I genuinely forgot.  I guess that until I can find an join a team that uses 80, I can comply.

File: modules/support-installing-packages-to-a-toolbox-container.adoc (hunk 2 above)
Comment (mikemckiernan) [reply modules/support-installing-packages-to-a-toolbox-container.adoc:10]: Huge thank you.  I don't have all the ISG details memorized and wouldn't have even thought to look this one up.  Sincere thanks.

File: support/gathering-cluster-data.adoc (hunk 3 above)
Comment (mikemckiernan) [reply support/gathering-cluster-data.adoc:51]: My instinct is that the `toolbox` command is overwhelmingly an intermediate step for troubleshooting.  I want the info available to customers, but not so visible as to make customer believe they need much knowledge of it.

Comment (codyhoag) [reply support/gathering-cluster-data.adoc:51]: Makes sense to me; thanks for explaining

File: modules/about-toolbox.adoc (hunk 1 above)
Comment (codyhoag) [reply modules/about-toolbox.adoc:9]: I had always hardwrapped at 80 before joining Red Hat. It still feels like I'm committing a crime when there's a 500 char single line :sweat_smile:

**Comment by codyhoag** [comment]
All updates LGTM. Merging...

**Pull request merged**